frontend/src/components/AudioPlayerLoading/index.jsx

```diff
@@ -67,43 +78,40 @@ export default function AudioPlayerLoading() {
 
   return (
     <SAudioPlayerLoading>
-      <section className="contain">
-        <div>{timer}</div>
-        <div>{duration}</div>
-      </section>
-      <section className="containInput">
-        <input
-          readOnly="readOnly"
-          className="range"
-          type="range"
-          min="0"
-          max={maxDuration}
-          value={timer}
-        />
-      </section>
+      <ThemeProvider theme={theme}>
+        <>
```

Comment:
So... we have a styled component wrapping a MUI theme provider wrapping a useless Fragment ? é_è